packages/eslint-config-javascript/eslint.config.js

```diff
@@ -0,0 +1,13 @@
+import javascript from '@ni/eslint-config-javascript';
+
+export default [
+    ...javascript,
+    {
+        files: ['**/*.js'],
+        rules: {
```

Comment:
It looks like these are new rule configurations for this project's source, is that correct? Can you comment why they were needed? Assuming there's a reasonable justification, i general it's nice to add a short code comment justifying deviations from the standard ruleset.

Similar question for other packages too.

Reply:
I took a pass at cleaning this up.
1. for `no-extraneous-dependencies` I was able to fix all the issues by either adding an explicit peerDependency in package.json or changing the import to be relative (in the case of the Angular package). https://github.com/ni/javascript-styleguide/pull/170/commits/8d0101324c65616d33145bb4b7d08888ef5e1c1e
2. for `no-unresolved` I found some links that describe this as a known issue so I narrowed the suppression to target just the typescript-eslint package https://github.com/ni/javascript-styleguide/pull/170/commits/f22f19ffe9ff8a0c4feabeb710aa4ad913a26ad6
3. I left `extensions` suppressed since our rule instructions say that's something projects might want to do